web/styles/settings.css

```diff
@@ -489,6 +489,32 @@ input[type="checkbox"] {
     }
 }
 
+#customizable_stream_notifications_table .dropdown-widget-button {
+    /* 225px at 16px/1em */
+    width: 14.0625em;
+
+    &:focus {
+        outline: none;
+    }
+
+    &:focus-visible {
+        outline: 2px solid var(--color-outline-focus);
+    }
+
+    .dropdown_widget_value {
+        display: flex;
+        align-items: baseline;
+        max-width: calc(100% - 1em);
+        gap: 0.4ch;
+
+        .decorated-channel-name {
+            overflow-x: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
```

Comment:
As, you said we can probably create a shared class and use that CSS for all dropdown widgets with streams.